etna/ciim/tests/test_models.py

```diff
@@ -329,7 +364,7 @@ def test_record_with_origination_but_no_date(self):
         record = create_record(
             iaid="C123456",
         )
-        del record["_source"]["@origination"]["date"]
+        del record["_source"]["origination"]["date"]
```

Comment:
@TNA-Allan has this changed for good in the API?

Reply:
The response now received is without @ symbol.